src/gobimport/validator/__init__.py

```diff
@@ -286,10 +340,15 @@ def _between_check(self, between, value):
         return value >= between[0] and value <= between[1] if value is not None else False
 
     def _geometry_check(self, between, value):
-        coords = re.findall('([0-9]+\.[0-9]{1,2})', value)
-        for count, value_range in enumerate(between):
-            # If the coord is outside of the supplied range, return False
-            if float(coords[count]) <= value_range[0] or float(coords[count]) >= value_range[1]:
+        coords = re.findall('([0-9]+\.[0-9]+)', value)
+        # Loop through all coords and check if they fill within the supplied range
+        # Even coords are x values, uneven are y values
+        coord_types = ['x', 'y']
+        for count, coord in enumerate(coords):
+            # Get the coord type
+            coord_type = coord_types[count % 2]
+            # If the coord is outside of the boundaries, retun false
+            if float(coord) < between[coord_type]['min'] or float(coord) > between[coord_type]['max']:
```

Comment:
not (between[coord_type]['min'] <= float(coord) and float(coord) <= between[coord_type]['max']))
reads a little more like testing if something is within range x in [a, b]
Question of taste